app/src/lib/open-shell.ts

```diff
@@ -15,14 +15,20 @@ export function openShell(fullPath: string, shell?: string) {
     return spawn('START', [shell || 'cmd'], { shell: true, cwd: fullPath })
   }
 
+  if (__LINUX__) {
+    const commandArgs = ['--working-directory', fullPath]
+    return spawn('gnome-terminal', commandArgs, { shell: true })
+  }
+
   return fatalError('Unsupported OS')
 }
 
 export function isGitOnPath(): Promise<boolean> {
   // Modern versions of macOS ship with a Git shim that guides you through
   // the process of setting everything up. We trust this is available, so
   // don't worry about looking for it here.
-  if (__DARWIN__) {
+  // I decide linux user have git too :)
```

Comment:
As you can see [here](http://releases.ubuntu.com/16.04/ubuntu-16.04.2-desktop-amd64.manifest), `git` isn't installed by default on Ubuntu 16.04.

I'm also very confident it's not installed by default on Fedora either.